src/ApimDeveloper/SFA.DAS.ApimDeveloper.UnitTests/Application/ApiSubscriptions/Commands/WhenHandlingDeleteSubscriptionKeyCommand.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture.NUnit3;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApimDeveloper.Application.ApiSubscriptions.Commands.RenewSubscriptionKey;
+using SFA.DAS.ApimDeveloper.Configuration;
+using SFA.DAS.ApimDeveloper.InnerApi.Requests;
+using SFA.DAS.ApimDeveloper.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.ApimDeveloper.UnitTests.Application.ApiSubscriptions.Commands
+{
+    public class WhenHandlingRenewSubscriptionKeyCommand
+    {
+        [Test, MoqAutoData]
+        public async Task Then_The_Api_Is_Called(
+            RenewSubscriptionKeyCommand command,
+            [Frozen] Mock<IApimDeveloperApiClient<ApimDeveloperApiConfiguration>> mockApiClient,
+            RenewSubscriptionKeyCommandHandler handler)
```

Comment:
I think something has got confused with this file and the one below, seem to have the wrong file name 🕺🏻

Reply:
Resovled the file name conflict